docker-entrypoint.sh

```diff
@@ -20,8 +20,12 @@ if [ $DEPLOYMENT_MODE == "LOCAL" ]; then
   >&2 echo "Postgres is up - executing command"
 fi
 
-# Run Data Migrations Only (Static files can remain local)
-echo "Running Data Migrations"
+# makes migrations, with merge flag to handle conflics
+echo "making Schema Migrations"
+python ./joplin/manage.py makemigrations --merge --noinput
```

Comment:
I'm worried that this is a little too aggressive. One main concern is if we made some changes while running locally and everything seems fine, but then forgot to shut down and restart the container. Then the migration might get made in a deployed environment and not checked in to code. Later on, it'd be possible to have our migrations out of sync between deployed environments and our codebase.

It's possible I'm overthinking this, but right now this seems scary.

Reply:
I get that line of thinking, it might be best if we could try to actually create that scenario and see what happens. 

Another thing I was thinking is that we could run 
`python ./joplin/manage.py makemigrations --check`
Which checks if there are model changes that have no migrations for them. We could put that in CI testing or also in this entrypoint?

The reason why I left it out was mainly thinking 
'What would we want to do if there weren't migrations made for a model change?'
and I couldn't think of cases where we wouldn't then want to make a migration, even if we ended up later committing a new migration that undid that change.

Reply:
@Niicck and I made this less scary!